README.md

```diff
@@ -1,22 +1,32 @@
 # Azure SDK for Rust
 
-This repository is for the development of the [unofficial](https://github.com/Azure/azure-sdk-for-rust/blob/main/FAQ.md#why-is-it-unofficial) Azure SDK for Rust.
+This repository is for the active development of the Azure SDK for Rust. For consumers of the SDK you can follow the links below to visit the documentation you are interested in
 
-## Crates
+* [Overview of Azure SDK for Rust](https://docs.microsoft.com/azure/developer/rust/)
+* [SDK Reference](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk)
+* [Code Samples for Azure SDK for Rust](https://github.com/azure-samples/azure-sdk-for-rust-samples)
+* [Azure REST API Docs](https://docs.microsoft.com/rest/api/)
+* [General Azure Docs](https://docs.microsoft.com/azure)
+* [Share your feedback to our Azure SDK](https://www.surveymonkey.com/r/FWPGFGG)
+
+## Getting Started
+
+To get started with a crate, see the README.md file located in the crate's project folder.  You can find these crate folders grouped by service in the `/sdk` directory.
+
+## Crates available
+
+Each service can have both 'client' and 'management' crates. 'Client' crates are used to consume the service, whereas 'management' crates are used to configure and manage the service.
```

Comment:
It may be first in a sentence, but the casing is specific. You can fix it with a colon.
```suggestion
Each service can have both 'client' and 'management' crates: 'client' crates are used to consume the service, whereas 'management' crates are used to configure and manage the service.
```